crates/store/re_grpc_client/src/message_proxy/read.rs

```diff
@@ -10,19 +10,18 @@ use crate::TonicStatusError;
 use crate::MAX_DECODING_MESSAGE_SIZE;
 
 pub fn stream(
-    endpoint: re_uri::ProxyEndpoint,
+    url: re_uri::ProxyEndpoint,
```

Comment:
We're now calling it three different things at the same time: `url`, `uri`, and `endpoint`. Can we decide on just one and rename the other parts to that? e.g. If we keep `re_uri`, then:
```suggestion
    uri: re_uri::ProxyUri,
```

Reply:
I agree completely… and really these are URLs, since they include scheme…

Reply:
I'll clean this up in a follow-up PR